src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,7 +20,10 @@
 public class RobotContainer {
   // Subsystems
   public final ManipulatorSubsystem manipulator = new ManipulatorSubsystem();
-  public final MecanumDrive drive = new MecanumDrive();
+
+  // one of the two following lines of code must be commented out at all times
+  public final DiferentialDrive drive = new DiferentialDrive();
```

Comment:
Let's just switch to using Mecanum for now

Reply:
switched to mecanum